Over21/ViewController.swift

```diff
@@ -128,6 +139,70 @@ extension ViewController: CaptureHelperDevicePresenceDelegate {
         print("scanner arrived")
         ageIndicatorView.updateScannerConnection(isConnected: true)
         
+        getDataSource(with: device) { [weak self] (result) in
+            guard let strongSelf = self else { return }
+            
+            if result == SKTResult.E_NOTSUPPORTED {
+                DispatchQueue.main.async {
```

Comment:
We should not require the Dispatch main queue here

Reply:
At first, I did not dispatch to main queue, but removing this will throw a Main-thread warning.
I understand that I set `main` as the `delegateDispatchQueue` for capture, but the completionHandler of the `getDataSource` function is not coming from Capture. I assume is the reason the main-thread warning occurs if I don't manually switch to the main thread.